test/testsupport/testdata/TdJsonBodies.scala

```diff
@@ -60,8 +60,7 @@ object TdJsonBodies {
       |      }
       |    },
       |    "sessionId": "IamATestSessionId",
-      |    "correlationId": "8d89a98b-0b26-4ab2-8114-f7c7c81c3059",
-      |    $jsonFormatted
+      |    "correlationId": "8d89a98b-0b26-4ab2-8114-f7c7c81c3059"$jsonFormatted
```

Comment:
is this correct? looks like a comma was dropped

Reply:
There's a check on journeyInfo list, if non empty add a comma